fixtures/make-travis-yml.travis.yml

```diff
@@ -0,0 +1 @@
+../.travis.yml
```

Comment:
First fixture is this repo itself. we could `cabal new-run golden -- --accept` to generate new `.travis.yml` (almost!) :)